modules/cert-manager-acme-dns01-explicit-gcp.adoc

```diff
@@ -4,17 +4,17 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="cert-manager-acme-dns01-explicit-gcp_{context}"]
-= Configuring an ACME issuer by using explicit credentials for GCP Cloud DNS
+= Configuring an ACME issuer by using explicit credentials for {gcp-short} Cloud DNS
```

Comment:
I wonder about "Google Cloud Cloud DNS" :thinking: The linked docs go to a page for "Google CloudDNS"